lib/src/Uncertainty/Model/openturns/DistributionImplementation.hxx

```diff
@@ -253,14 +253,16 @@ public:
 
   /** Get the probability content of an interval */
   virtual Scalar computeProbability(const Interval & interval) const;
-protected:
   /** Generic implementation for continuous distributions */
+  virtual Scalar computeProbabilityGeneral(const Interval & interval) const;
+  /** Generic implementation for 1D continuous distributions */
+  virtual Scalar computeProbabilityGeneral1D(const Scalar a, const Scalar b) const;
+  /** Generic implementation for continuous distribution by integration of the PDF */
   virtual Scalar computeProbabilityContinuous(const Interval & interval) const;
+  /** Generic implementation for 1D continuous distribution by integration of the PDF */
+  virtual Scalar computeProbabilityContinuous1D(const Scalar a, const Scalar b) const;
   /** Generic implementation for discrete distributions */
   virtual Scalar computeProbabilityDiscrete(const Interval & interval) const;
-  /** Generic implementation for general distributions */
-  virtual Scalar computeProbabilityGeneral(const Interval & interval) const;
-public:
 
```

Comment:
so all these are public now ?

Reply:
@jschueller Yes, I need to be able to call them from other distributions such as KernelMixture in order to avoid the costly creation/destruction of small objects.